### PR TITLE
chore: Remove crufty CRB code

### PR DESCRIPTION
**Pull request by uchukwu**
#### Summary of changes

**Asana Ticket:** [💳 Remove crufty CRB code](https://app.asana.com/0/584764604969369/1202148457379660/f)

- In TrainLoc.Manager.handle_events all the code in the handle_old_event code path is removed.
- Removed associated unit tests

#### Reviewer Checklist

- [ ] Meets ticket's acceptance criteria
- [ ] Tests were added for any new functionality
- [ ] [GTFS documentation](https://github.com/mbta/gtfs-documentation/) has a PR (if data is added/changed)
- [ ] Relevant stakeholders are notified (if data is changed)
- [ ] Changes are verified on schedule_viewer or PartnerDash

### CODE REVIEW

File: apps/train_loc/test/integration/integration_test.exs [hunk 1]
```diff
@@ -68,9 +68,4 @@ defmodule TrainLoc.IntegrationTest do
         :timeout
     end
   end
-
-  @tag :integration
-  test "one minute of messages with 11 up-to-date vehicles and one conflict" do
-    run_test(TrainLoc.IntegrationTest.Scenarios.OneMinute)
```
Comment (paulswartz) [inline apps/train_loc/test/integration/integration_test.exs:74]: question: what module is this referencing? can we delete that module as well?

Comment (uchukwu) [reply apps/train_loc/test/integration/integration_test.exs:74]: Removed `Scenarios.OneMinute`. That was only referenced from the removed IntegrationTest unit test.

**Review by paulswartz — APPROVED** [review]
🍰

**Comment by uchukwu** [comment]
Test

**Pull request merged**